src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rent = 40;
+  const rentReduce3 = 20;
+  const rentReduce7 = 50;
+
+  if (days < 3) {
+    return days * rent;
+  } else if (days >= 3 && days < 7) {
```

Comment:
```suggestion
  } else if (days < 7) {
```
Якщо перша перевірка не пройшла, то значить днів точно >=3. Отже звідси це можна прибрати